docs/guide/ops/catalog/index.md

```diff
@@ -284,6 +291,19 @@ When referencing a blueprint, if a version number is not specified
 the latest non-snapshot version will be loaded when an entity is instantiated.
 
 
+### CLI Options
+
+The `brooklyn` CLI includes several commands for working with the catalog.
+
+* `--catalogAdd <file.bom>` will add the catalog items in the `bom` file
```

Comment:
i expect we'll be adding a lot more, explaining this, and converting `*.yaml` to `*.bom` in near future